src/main/resources/texts/texterMU_LUAE_NA_v1.2.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<texter xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" giltigFrom="2022-07-05" typ="LUAE_NA" version="1.2" xsi:schemaLocation="http://schema.forsakringskassan.se/vardgivare_sjukvard/texterMU_v1.0.xsd" xmlns="http://schema.forsakringskassan.se/vardgivare_sjukvard">
```

Comment:
@IamFriberg  @mww-sopra-steria Reading jira 14704 it seems that the official release date for the new text version is 2022-09-22. Perhaps the date i prod version should be set to reflect this? (It will make no functional difference but to be correct?).

Reply:
Fixed